lib/connectors/memory.js

```diff
@@ -567,9 +567,9 @@ function applyFilter(filter) {
         return compare(example.neq, value) !== 0;
       }
 
-      if ('between' in example ) {
-        return ( testInEquality({gte:example.between[0]}, value) &&
-                 testInEquality({lte:example.between[1]}, value) );
+      if ('between' in example) {
+        return (testInEquality({ gte:example.between[0] }, value) &&
+                 testInEquality({ lte:example.between[1] }, value));
```

Comment:
Just out of curiosity if we break a line to multiple lines, what is the preferred style?

Based on `Eslint --fix` it seems it adds 9 spaces to indent the second line; should it be two spaces?
```
return (testInEquality({ gte:example.between[0] }, value) &&
  testInEquality({ lte:example.between[1] }, value));
```

Just wanted to know the preference in the team; also the proceeding lines do not have any more indentation, I believe, right?

Example:
```
return (testInEquality({ gte:example.between[0] }, value) &&
  testInEquality({ lte:example.between[1] }, value) &&
  testInEquality({ lte:example.between[2] }, value));
```